### DISCUSSION TITLE
Problem with working directory in included .Rmd files

### DISCUSSION THREAD

**Discussion by aloboa** [quarto-dev/quarto-cli#10184] [Q&A]
### Description

Coming from rmarkdown, I like to save time by including existing .Rmd files within my new .qmd files (eg, Qmain.qmd) , which I can do with either:

`{{< include "../HandSamples/FS3_MineralCompo/test.Rmd" >}}`

or even as in rmarkdown:
{r child = '../HandSamples/FS3_MineralCompo/test.Rmd'}
(cannot reproduce the 3 quotes here)

The problem is that the included test.Rmd file is in a different  directory, but it is executed as if its working directory
were the same as the one from which it is called (the one of Qmain.qmd), which creates problems with relative paths in test.Rmd.

Is there any way to force that test.Rmd be executed on its own directory?

**Comment by mcanouil** [comment]
As described in the documentation, the include shortcode is a copy/paste, so what matters is the main document.

You can look at the "embed" shortcode instead.

Regarding the three backticks, the process/syntax is described above the text field when you open a discussion, i.e., in markdown to include backticks you need to enclose them with more.
